src/DataWrangling/metadata.jl

```diff
@@ -178,3 +178,59 @@ function compute_native_date_range(native_dates, start_date, end_date)
 
     return native_dates[start_idx:end_idx]
 end
+
+"""
+    z_faces(metadata)
+
+Return an array with the ``z``-faces of the dataset that `metadata` corresponds to.
+"""
+z_faces(metadata::Metadata{V}) where V =
+    error("z_faces not implemented for $V")
+
+variable_is_three_dimensional(metadata::Metadata{V}) where V =
+    error("variable_is_three_dimensional not implemented for $V")
+
+"""
+    empty_field(metadata::Metadata;
+                architecture = CPU(),
+                horizontal_halo = (7, 7))
+
+Return an empty field of `metadata` on `architecture` and with `horizontal_halo`s.
+"""
+function empty_field(metadata::Metadata;
+                     architecture = CPU(),
+                     horizontal_halo = (7, 7))
+
+    Nx, Ny, Nz, _ = size(metadata)
+    loc = location(metadata)
+    longitude = (0, 360)
+    latitude = (-90, 90)
+    TX, TY = (Periodic, Bounded)
+
+    if variable_is_three_dimensional(metadata)
+        TZ = Bounded
+        LZ = Center
+        z = z_faces(metadata)
+        halo = (horizontal_halo..., 3)
+        sz = (Nx, Ny, Nz)
+    else # the variable is two-dimensional
+        TZ = Flat
+        LZ = Nothing
+        z = nothing
+        halo = horizontal_halo
+        sz = (Nx, Ny)
+    end
+
+    grid = LatitudeLongitudeGrid(architecture, Float32; halo, longitude, latitude, z,
+                                 size = sz,
+                                 topology = (TX, TY, TZ))
+
+    return Field{loc...}(grid)
+end
+
+shift_longitude_to_0_360(data, metadata) = data
+
+struct Celsius end
+struct Kelvin end
+
+dataset_temperature_units(metadata) = nothing
```

Comment:
are these valid units? I thikn if there is not a good fallback then it is better not to define it. Once can simply create a function with no methods by writing

```julia
function dataset_temperature_units end
```

Reply:
good point!